app/pb_kits/playbook/pb_typeahead/_typeahead.jsx

```diff
@@ -45,14 +51,27 @@ const Typeahead = (props: Props) => {
     },
     isClearable: true,
     isSearchable: true,
+    name,
     ...props,
   }
 
+  if (typeof(props.loadOptions) === 'string') selectProps.loadOptions = get(window, props.loadOptions)
```

Comment:
If we detect that we have a string here, we can assume that this means a variable in the `window` scope and that we should attempt to grab that and use it.